src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -10,6 +10,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(TargetFrameworkIdentifier)' == '.NETFramework' ">
```

Comment:
this is not needed, MSBuild does it for us. We are already using `#if !NETFRAMEWORK` in many places: https://github.com/dotnet/performance/search?q=NETFRAMEWORK&unscoped_q=NETFRAMEWORK

Please remove it

Reply:
Done